### PR TITLE
fix(table.js): moved  thead into table component

### PR DESCRIPTION
**Pull request by gautism1**

### CODE REVIEW

File: src/components/Table.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import PropTypes from "prop-types";
-function Table({ tableData, formatter }) {
+function Table({ tableData, formatter, children }) {
```
Comment (InvincibleZeal) [inline src/components/Table.js]: take remaining props as `...rest` and spread them on the`<table>` element

```js
function Table({ tableData, formatter, children, ...rest}) {
```

File: src/components/Table.js [hunk 2]
```diff
@@ -44,14 +44,7 @@ function Table({ tableData, formatter }) {
     return (
         <>
             <table className="table px-5">
```
Comment (InvincibleZeal) [inline src/components/Table.js]: spread the rest props 
```jsx
            <table {...rest}>
````
remove `className` from here and add it to `Table` components in ListCustomer, ListItems and ListInvoices.

**Review by InvincibleZeal — APPROVED** [review]

File: src/components/Table.js [hunk 3]
```diff
@@ -38,14 +42,25 @@ function Table({ tableData, formatter }) {
         return body;
     };
     return (
-        <tbody>
-            <TableBody />
-        </tbody>
+        <>
```
Comment (rparsh) [inline src/components/Table.js:45]: why using this?

File: src/components/Table.js [hunk 4]
```diff
@@ -1,8 +1,12 @@
 import React from "react";
 import PropTypes from "prop-types";
-function Table({ tableData, formatter }) {
+function Table({ tableData, formatter, children, ...rest }) {
```
Comment (rparsh) [inline src/components/Table.js:3]: there will be no children support here. remove it

File: src/components/Table.js (hunk 3 above)
Comment (gautism1) [reply src/components/Table.js:45]: removed,

**Review by rparsh — APPROVED** [review]

**Pull request merged**